src/server/system_services/bucket_server.js

```diff
@@ -1623,11 +1613,13 @@ function _calc_metrics({
     let is_quota_exceeded = false;
     let is_quota_low = false;
     if (bucket.quota) {
-        let quota_precent = system_utils.get_bucket_quota_usage_percent(bucket, bucket.quota);
-        info.quota = _.omit(bucket.quota, 'value');
-        if (quota_precent >= 100) {
+        let quota = new Quota(bucket.quota);
```

Comment:
Why aren't we starting the Quota class in the upper scope? 
We are doing `new Quota(bucket.quota); ` here and in line 653 (get_bucket_changes_quota)

Reply:
these are different requests